CONTRIBUTING.rst

```diff
@@ -69,37 +119,37 @@ For a more detailed discussion, read these :doc:`detailed documents
-      write inline and/or general comments on your Pull Request (PR) to help
-      you improve its implementation, documentation, and style.  Every single
-      developer working on the project has their code reviewed, and we've come
-      to see it as friendly conversation from which we all learn and the
-      overall code quality benefits.  Therefore, please don't let the review
-      discourage you from contributing: its only aim is to improve the quality
-      of project, not to criticize (we are, after all, very grateful for the
-      time you're donating!).
+   * Reviewers (the other developers and interested community members) will
+     write inline and/or general comments on your Pull Request (PR) to help
+     you improve its implementation, documentation, and style.  Every single
+     developer working on the project has their code reviewed, and we've come
+     to see it as friendly conversation from which we all learn and the
+     overall code quality benefits.  Therefore, please don't let the review
+     discourage you from contributing: its only aim is to improve the quality
+     of project, not to criticize (we are, after all, very grateful for the
+     time you're donating!).
 
-    * To update your pull request, make your changes on your local repository
-      and commit. As soon as those changes are pushed up (to the same branch as
-      before) the pull request will update automatically.
+   * To update your pull request, make your changes on your local repository
+     and commit. As soon as those changes are pushed up (to the same branch as
+     before) the pull request will update automatically.
 
-    * `Travis-CI <https://travis-ci.org/>`_, a continuous integration service,
-      is triggered after each Pull Request update to build the code and run unit
-      tests of your branch. The Travis tests must pass before your PR can be merged.
-      If Travis fails, you can find out why by clicking on the "failed" icon (red
-      cross) and inspecting the build and test log.
+   * `Travis-CI <https://travis-ci.org/>`_, a continuous integration service,
+     is triggered after each Pull Request update to build the code and run unit
+     tests of your branch. The Travis tests must pass before your PR can be merged.
+     If Travis fails, you can find out why by clicking on the "failed" icon (red
+     cross) and inspecting the build and test log.
 
-    * `AppVeyor <http://ci.appveyor.com>`_, is another continuous integration
-      service, which we use.  You will also need to make sure that the AppVeyor
-      tests pass.
-
-.. note::
+   * `AppVeyor <http://ci.appveyor.com>`_, is another continuous integration
+     service that we use.  You will also need to make sure that the AppVeyor
+     tests pass.
 
-   If closing a bug, also add "Fixes #1480" where 1480 is the issue number.
+   .. note::
+   
+      If closing a bug, also add "Fixes #1480" where 1480 is the issue number.
```

Comment:
Github seems to have added a "Linked Issues" feature under the Milestone setting options in the right side-panel on the PR page. I have used it but haven't tested it much. It might make "Fixes" obsolete.  

Also, we should say **where** to add "Fixes #1480".  It seems that adding it to a PR comment is insufficient. It must appear in the commit message for the merged (squashed and merged) action.  I'm thinking of wording like:
```
"Fixes #1480" should be added to a commit message where 1480 is the issue number
```

Reply:
I went with:

>    If the PR closes an issue, make sure that GitHub knows to automatically
      close the issue when the PR is merged.  For example, if the PR closes
      issue number 1480, you could use the phrase "Fixes #1480" in the PR
      description or commit message.